packages/x-components/src/x-modules/search/store/__tests__/getters.spec.ts

```diff
@@ -18,52 +16,24 @@ describe('testing search module getters', () => {
   });
 
   describe(`${gettersKeys.request} getter`, () => {
-    it('should return a request object if there is a query', () => {
+    it('should return a request object if there is a query with module properties', () => {
       resetSearchStateWith(store, {
         query: 'doraemon',
         params: {
           catalog: 'es'
         }
       });
 
-      expect(store.getters[gettersKeys.request]).toEqual<SearchRequest>({
+      expect(store.getters[gettersKeys.request]).toEqual<PageableSearchRequest>({
         query: 'doraemon',
         relatedTags: [],
         filters: {},
         sort: '',
-        rows: 24,
-        start: 0,
+        page: 1,
```

Comment:
should we add another test for pages bigger than one?

Reply:
Done